src/aiida/cmdline/commands/cmd_computer.py

```diff
@@ -62,14 +62,19 @@ def _computer_test_no_unexpected_output(transport, scheduler, authinfo, computer
     This can happen if e.g. there is some spurious command in the
     .bashrc or .bash_profile that is not guarded in case of non-interactive
     shells.
+    Note: This test is irrelevant if the transport plugin does not support command execution.
 
     :param transport: an open transport
     :param scheduler: the corresponding scheduler class
     :param authinfo: the AuthInfo object (from which one can get computer and aiidauser)
     :return: tuple of boolean indicating success or failure and an optional string message
     """
-    # Execute a command that should not return any error
-    retval, stdout, stderr = transport.exec_command_wait('echo -n')
+    # Execute a command that should not return any error, except NotImplementedError (non-ssh transports)
```

Comment:
```suggestion
    # Execute a command that should not return any error, except ``NotImplementedError``
    # since not all transport plugins implement remote command execution.
```